webapp/WEB-INF/templates/admin/plugins/workflow/modules/formspdf/include/formstopdf-commons.html

```diff
@@ -0,0 +1,58 @@
+<#macro marker_descriptions>
+<h3>#i18n{module.workflow.formspdf.markers.label_table_title}</h3>
+<@table>
+	<tr>
+		<th>#i18n{module.workflow.formspdf.markers.label_description}</th>
+		<th>#i18n{module.workflow.formspdf.markers.label_marker}</th>
+		<th></th>
+	</tr>
+	<#if list_markers?exists>
+		<#list list_markers as marker>
+		<tr>
+			<@td class='marker'>
+				<#if marker.description ??>
+					<#assign description = i18n(marker.description) />
+					<#if description?has_content>${description}<#else>${marker.description!''}</#if>
+				</#if>	
+			</@td>
+			<@td class='marker'>&#36;&#123;${marker.marker}&#33;&#125;</@td>
+			<@td><@icon style='copy' title='Copy line' /></@td>
+		</tr>
+		</#list>
+	<#else>
+		<tr>
+			<@td colspan=3></@td>
+		</tr>
+	</#if>
+</@table>
+<style>
+.marker:hover, .fa-copy:hover{ cursor: pointer; opacity: .8}
+.marker:nth-child(even) td{ background: #777; color: #fff }
+.copied::after{ content:' Ligne copiée !' }
+</style>
+<script>
+function copyToClipboard( txt ) {
+  var $temp = $("<input>"), sTxt=txt.replace(/\s+/g, ' ');
+  $("body").append($temp);
+  $temp.val( sTxt ).select();
+  document.execCommand("copy");
+  $temp.remove();
+}
+
+$('.marker').on( 'dblclick', function(e){
+	if( $( '.tab-pane.active .richtext' ).length > 0 ){
+		var c = '<p>' + $(this).text() + '</p>';
+		tinymce.activeEditor.insertContent( '<p>' + c + '</p>' );
```

Comment:
warning : the rich text editor could be another one than  tinymce.
Consult this issue to use a generic function : 
https://dev.lutece.paris.fr/bugtracker/issues/26580

Reply:
Thanks for your comments ! 
Is the double-click function crucial for this ticket or not ? 
Because in my last commit I've outdated the double-click function, since the copy & paste now performed correctly and the PR is getting older.